### PR TITLE
Workroles checkboxes

### PR DESCRIPTION
**Pull request by jiiaa**
Now a user can select the preferable roles by clicking the checkboxbuttons instead of inputing something of his/her own. This makes the matching with open positions easier and more consistent.

### CODE REVIEW

File: front/src/Containers/ConsultantList.js [hunk 1]
```diff
@@ -16,15 +17,18 @@ class ConsultantList extends Component {
...
-      searchTerm: ""
+      noServerResponse: false,
+      searchTerm: "",
     };
   }
 
   componentDidMount = () => {
     GetAllConsultants(response => {
-      if (response.status === 200) {
+      if (!response)
```
Comment (rrudd) [inline front/src/Containers/ConsultantList.js]: missing {}

File: front/src/Containers/ConsultantList.js [hunk 2]
```diff
@@ -59,8 +63,12 @@ class ConsultantList extends Component {
 
     return (
       <Container>
-        <SearchInput onChange={this.searchUpdated} className="search-input" />
-        {consultantsListed}
+        { this.state.noServerResponse ? <Redirect to="/error" />
+        : <>
+          <SearchInput onChange={this.searchUpdated} className="search-input" />
```
Comment (rrudd) [inline front/src/Containers/ConsultantList.js]: indentation

File: front/src/Containers/UserProfile.js [hunk 3]
```diff
@@ -18,55 +20,67 @@ export default class UserProfile extends Component {
...
+  */
+
+  getUserInfoRole = async () => {
     const userRole = sessionStorage.getItem('aw-role');
-    console.log("userRole from sessionStorage: ", userRole);
-    GetConsultantInfobyEmail(this.state.userEmail, response => {
-      if (response.status === 200) {
-        let user = response.data;
-        let userIsConsultant = userRole !== "AW";
+    console.log("userRole: ", userRole);
```
Comment (rrudd) [inline front/src/Containers/UserProfile.js]: remove console logs

File: front/src/Routes.js [hunk 4]
```diff
@@ -1,25 +1,27 @@
...
+import ErrorPage from './Components/ErrorPage';
+import Login from './Views/Login';
 import PositionsList from "./Containers/PositionsList";
 import PositionSelection from "./Containers/PositionSelection";
-import AddNewPosition from "./Containers/AddNewPosition";
-import Login from './Views/Login';
-import ConsultantList from './Containers/ConsultantList';
+import ProfileView from "./Views/ProfileView";
+import ViewNonEditableProfile from "./Containers/ViewNonEditableProfile";
+import "bootstrap/dist/css/bootstrap.min.css";
```
Comment (rrudd) [inline front/src/Routes.js]: do we need this?

File: front/src/Components/Styles/App.css [hunk 5]
```diff
@@ -85,14 +85,44 @@ span.small-card-skill-badge.badge.badge-secondary {
...
+  margin: 3px;
   background-color: #047364;
 }
 
+button.btn.btn-primary {
+  color: #FFFFFF;
+  background-color: rgb(175, 180, 185);
+  font-size: 11px;
+  font-weight: bold;
+  margin: 3px !IMPORTANT;
```
[the remark this answers is not in the document]
Comment (jiiaa) [reply front/src/Components/Styles/App.css]: One !IMPORTANT left which is for the ACTIVE checkboxbutton.

**Pull request merged**